### PR TITLE
[easy] Fix docs for sd calculation in BatchNorm1d/3d for consistency with BatchNorm2d

### PR DESCRIPTION
**Pull request by mikaylagawarecki**
Fixes https://github.com/pytorch/pytorch/issues/100048

BatchNorm2d docs were updated in https://github.com/pytorch/pytorch/pull/97974. There have been a number of issues filed due to confusion about this so I think we should fix before branch cut

Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #107308

### CODE REVIEW

**Review by albanD — APPROVED** [review]
Good catch
